src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -171,10 +220,14 @@ export function useSwapInputsController({
       data,
       outputAmount,
       inputAmount,
+      inputPrice,
+      outputPrice,
     }: {
       data: Quote | CrosschainQuote | QuoteError | null;
       outputAmount?: number;
       inputAmount?: number;
+      inputPrice?: number | null;
+      outputPrice?: number | null;
     }) => {
       'worklet';
 
```

Comment:
for discussion: we're doing this slider update in multiple places - what if we turned the slider into an animated reaction and react when: the input amount changes and the input method is not the slider